supertokens_python/recipe/session/interfaces.py

```diff
@@ -250,7 +242,7 @@ async def set_claim_value(
         session_handle: str,
         claim: SessionClaim[_T],
         value: _T,
-        user_context: Dict[str, Any],
+        user_context: Optional[Dict[str, Any]] = None,
```

Comment:
userContext here should not be optional since it's part of recipe interface

Reply:
done